Gemfile

```diff
@@ -1,23 +1,18 @@
 source 'https://rubygems.org'
 
-gem 'addressable', '2.3.5'      # pin to avoid RDF bug
-gem 'dor-services', '~> 4.8'
+# General ruby dependencies
+gem 'pry', '~> 0.10.0'           # for bin/console
+gem 'slop', '~> 3.5.0'           # for bin/run_robot
+gem 'rake', '~> 10.3'
+
+# DLSS gems
+gem 'dor-services', '~> 5.3'
 gem 'lyber-core', '~> 3.2', '>=3.2.4'
```

Comment:
Curious about how to interpret the version spec on lyber-core here.  Will this spec allow it to get any 3.x greater than 3.2, say 3.3.x?